Cargo.lock

```diff
@@ -4160,14 +4160,14 @@ dependencies = [
  "futures-channel",
  "futures-core",
  "futures-util",
- "h2 0.3.22",
+ "h2 0.3.24",
  "http 0.2.9",
  "http-body",
  "httparse",
  "httpdate",
  "itoa",
  "pin-project-lite",
- "socket2 0.5.5",
```

Comment:
I'm curious why this caused `socket2` to be downgraded.

Reply:
It seems like `cargo update -p   h2@0.3.22` did it, though I can't quite figure out why since `cargo tree -p h2@0.3.24` shows a dependency on `socket@0.5.5`. Running `cargo update -p socket2@0.4.10` put it back. Thanks for catching that.